report-summary-merged-prs

```diff
@@ -1466,7 +1466,7 @@ if __name__ == "__main__":
     MAGIC_COMMAND_FIND_VALGRIND = 'test -d ' + JENKINS_ARTIFACTS_DIR + '/valgrind/RELEASE_NAME'
     MAGIC_COMMAND_FIND_IGPROF = 'test -d ' + JENKINS_ARTIFACTS_DIR + '/igprof/RELEASE_NAME'
     MAGIC_COMMAND_FIND_PROFILING = 'test -d ' + JENKINS_ARTIFACTS_DIR + '/profiling/RELEASE_NAME'
-    MAGIC_COMMAND_FIND_PROFILING_CHECKS_FILTER1 = 'test -s ' + JENKINS_ARTIFACTS_DIR + '/profiling/RELEASE_NAME/ARCHITECTURE/WORKFLOW/step3_RAW2DIGI_L1Reco_RECO_RECOSIM_PU.resources.json && echo ARCHITECTURE/WORKFLOW/step3_RAW2DIGI_L1Reco_RECO_RECOSIM_PU.resources'
+    MAGIC_COMMAND_FIND_PROFILING_CHECKS_FILTER1 = 'test -s ' + JENKINS_ARTIFACTS_DIR + '/profiling/RELEASE_NAME/ARCHITECTURE/WORKFLOW/step3_cpu.resources.json && echo ARCHITECTURE/WORKFLOW/step3_cpu.resources'
```

Comment:
@gartung , will this work for other release cycles where we have long wf names?

Reply:
Yes. The head of master in cmpwg/profiling was updated.

Reply:
If I use the *34.99 workflows the names of the steps are incorrect.

Reply:
Can I use a wildcard for the file name?

Reply:
yes you should be able to use wildcard but make sure that command properly returns `ARCHITECTURE/WORKFLOW/step3_*.resources` name

Reply:
I think something like the following should work
```
'test -s '+JENKINS_ARTIFACTS_DIR+'/profiling/RELEASE_NAME/ARCHITECTURE/WORKFLOW/step3_*.resources.json && (ls '+JENKINS_ARTIFACTS_DIR+'/profiling/RELEASE_NAME/ARCHITECTURE/WORKFLOW/step3_*.resources.json | sed "s|.*/RELEASE_NAME/||")`
```